aggregator/pom.xml

```diff
@@ -80,11 +72,6 @@
             <artifactId>quarkus-flyway</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-junit5</artifactId>
```

Comment:
did you remove it because it's included in the quarkus-junit5-mockito dependency already?

Reply:
Yes

Reply:
ok thx